app/Http/Controllers/BeatmapsController.php

```diff
@@ -86,4 +107,68 @@ public function scores($id)
             return error_popup($ex->getMessage());
         }
     }
+
+    /**
+     * Get a User Beatmap score
+     *
+     * Return a [User](#user)'s score on a Beatmap
+     *
+     * ---
+     *
+     * ### Response Format
+     *
+     * Returns [BeatmapUserScore](#beatmapuserscore)
+     *
+     * The position returned depends on the requested mode and mods.
+     *
+     * @urlParam beatmap required Id of the [Beatmap](#beatmap).
+     * @urlParam user required Id of the [User](#user).
+     *
+     * @queryParam mode The [GameMode](#gamemode) to get scores for.
+     * @queryParam mods An array of matching Mods, or none // TODO.
+     */
+    public function userScore($beatmapId, $userId)
+    {
+        $beatmap = Beatmap::findOrFail($beatmapId);
+        if ($beatmap->approved <= 0) {
+            abort(404);
+        }
+
+        $params = get_params(request()->all(), null, [
+            'mode:string',
+            'mods:any',
```

Comment:
`string[]` here and `array_filter` down there? Apparently `presence` doesn't always return string :eyes:  (also for the one in `#scores`)